extensions/vscode/snippets/bloc.json

```diff
@@ -1,25 +1,4 @@
 {
-  "Cubit": {
```

Comment:
why do you want to remove these? I would argue they're still useful in situations where you just want to create a new bloc/cubit in the existing file

Reply:
@felangel, mainly to use the `bloc` prefix for `context.bloc()`.

Reply:
We could keep the bloc and cubit class snippets, but I'd suggest adding a different prefix.